lib/ansible/roles/newrelic/tasks/main.yml

```diff
@@ -7,6 +7,12 @@
   apt_repository: repo='deb http://apt.newrelic.com/debian/ newrelic non-free' state=present
   sudo:           yes
 
+# if upgrading to 7 or downgrading to 5, remove & purge existing agent, let it install again with new php
+- name:           Remove and purge existing agent (when upgrading/downgrading PHP)
+  apt:            name=newrelic-php5 state=absent purge=yes
```

Comment:
Ensure we completely remove any vestiges of the nr agent when moving from one php version to another, so the normal install below will handle version and path differences